CDS/scripts/js_utils/datastore/config.js

```diff
@@ -0,0 +1,66 @@
+const path = require('path');
+const fs = require('fs');
+const PropertiesReader = require('properties-reader');
+
+class Config {
+    constructor (configDirectory = '/etc/cds_backend/conf.d', namespaceChar = '_') {
```

Comment:
a bit concerned that _ is fairly likely to be in a valid key name, so might this break/fail if there are too many _ in the string? e.g., meta_gnm_asset_category for example.